app/views/posts/index.html.erb

```diff
@@ -1,27 +1,21 @@
-<% stylesheet_link_tag "tables"%>
+<% stylesheet_link_tag "posts-list"%>
 
-<h1>Listing Posts</h1>
+<main class='main-container'>
+  <h1>Listing Posts</h1>
+  <%= render partial: "shared/user-container", locals: { user: @user } %>
 
-<table>
-  <thead>
-    <tr>
-      <th>Username</th>
-      <th>Post id</th>
-      <th>Title</th>
-      <th>Text</th>
-    </tr>
-  </thead>
-
-  <tbody>
+  <section class='posts-section'>
     <% @posts.each do |post| %>
-      <tr>
-        <td><%= @user.name %></td>
-        <td><%= post.id %></td>
-        <td><%= post.title %></td>
-        <td><%= post.text %></td>
-      </tr>
+      <%= render partial: 'shared/post-container', locals: { user:@user, post: post } %>
+      <div class='post-list-comments'>
+        <% @comments.each do |comment| %>
+          <% if comment.author_id == @user.id %>
+            <%= render partial: 'shared/comment-container', locals: { comment: comment, user: @user } %>
+          <% end %>
+        <% end %>
+      </div>
     <% end %>
-  </tbody>
-</table>
+  </section>
```

Comment:
As requested by the previous CR in the post index page, it should show the last 5 comments as shown in the wireframe, Currently, it is showing all the comments although it doesn't are from the current post.

| Wireframe | Your design |
|---|---|
|![image](https://github.com/dev89s/blog-app/assets/58642949/387c6818-1d71-4f0b-bc2c-e1be99544741) | ![image](https://github.com/dev89s/blog-app/assets/58642949/1dd90c5f-9527-48fc-b646-a8e23623e651) |